### DISCUSSION TITLE
Heyo, timer improvement for boat league- set to previous time, so we can set a 32/33 minute timer and have a simpler way to reset

### DISCUSSION THREAD

**Discussion by Nchi** [Lailloken/Exile-UI#470] [Ideas & Feedback]
Maybe you could shortcut it a bit by working off the set name- if it ends in 33m it resets to 33:00, etc., since bothering to cache something like that could be give or take lol.

**Comment by Lailloken** [comment]
Thanks for your suggestion. Since this is just a minor feature, I don't want to make it too complex by adding special rules/use-cases that get lost in documentation. I'll add a start and cancel button for the next release so times can be entered and confirmed with less hand movement (moving the left hand across the keyboard or letting go of the mouse in order to press ENTER).

You'll be able to quickly type ` 33 3 ` and click "start" to start a 33:03 timer.
![image](https://github.com/user-attachments/assets/01177284-f62d-4b29-adab-775a519069c9)
